schemas/code_signature.yml

```diff
@@ -57,3 +57,25 @@
         This is useful for logging cryptographic errors with the certificate validity or trust status.
         Leave unpopulated if the validity or trust of the certificate was unchecked.
       example: ERROR_UNTRUSTED_ROOT
+      
+    - name: team_id
+      level: extended
+      type: keyword
+      short: The team identifier used to sign the process.
+      description: >
+        The team identifier used to sign the process.
+
+        This is used to identify the team or vendor of a software product.
+        Leave unpopulated if unavailable.
```

Comment:
```suggestion
```

nit: This is the general guidance for ECS fields, so we can shorten the description by omitting.